Portfolio/index.css

```diff
@@ -1 +1,64 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+      color: #515050;
+          margin: 0%;
+    padding: 0%;
+}
+.cont1{
+    background-color: #90A4AE;
+    margin: 0%;
+    padding: 10px;
+  
+}
+nav ul{
+    padding: 0px;
+    margin: 0px;

```

Comment:
Consider adding a space between the property and value, like 'list-style-type: none;' for consistency